### PR TITLE
Simple Header 🧢

### PR DESCRIPTION
**Pull request by pierreericgarcia**

### CODE REVIEW

**Comment by pierreericgarcia** [comment]
[ETQ U connecté, je peux retourner à toutes mes demandes via l'accès rapide du header et j'ai un header minimisé en version connectée](https://trello.com/c/jPUFmZgS/1682-etq-u-connect%C3%A9-je-peux-retourner-%C3%A0-toutes-mes-demandes-via-lacc%C3%A8s-rapide-du-header-et-jai-un-header-minimis%C3%A9-en-version-connect%C3%A9)

File: frontend/src/components/organisms/Header.js [hunk 1]
```diff
@@ -76,34 +88,43 @@ const Header = () => {
...
+                    habilitations juridiques
+                  </p>
+                )}
               </div>
             </div>
             <div className="fr-header__tools">
               <div className="fr-header__tools-links">
                 <ul className="fr-btns-group">
-                  {displayContactLink && (
+                  {user && !user.roles.length && (
```
Comment (rdubigny) [inline frontend/src/components/organisms/Header.js:101]: we use the fonction isEmpty for a similar test here https://github.com/betagouv/datapass/blob/e3f17bae768d9624f9a1a8ef453b781376c8ff57/frontend/src/Routes.js#L67

We should use the same evalutation everywhere to ensure strange side effect on edge cases. Ex: if roles is null or undefined.

**Review by rdubigny — CHANGES_REQUESTED** [review]
I got a very short comment !
